frontends/election-manager/src/components/print_ballot_button_text.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  BallotLocales,
+  Election,
+  getElectionLocales,
+} from '@votingworks/types';
+import pluralize from 'pluralize';
+import { DEFAULT_LOCALE } from '../config/globals';
+import { BallotMode, ballotModeToReadableString } from '../config/types';
+import { getHumanBallotLanguageFormat } from '../utils/election';
+
+interface Props {
+  ballotCopies: number;
```

Comment:
Very minor: I like to name numerical props like this `numBallotCopies` or `numberOfBallotCopies` for clarity. Not a huge deal because we've already typed it as a number

Reply:
Going to leave this one as is, just to avoid changing variable names in other files.